content/source/docs/enterprise/api/modules.html.md

```diff
@@ -311,20 +372,33 @@ After the registry module version is successfully parsed by TFE, its status will
 
 ## Delete a Module
 
-These endpoints can delete a single version for a provider, a single provider (and all its versions) for a module, or a whole module. If the requested deletion would leave a provider with no versions or a module with no providers, the empty items will be automatically deleted as well.
+When removing modules, there are three versions of the endpoint, depending on how many parameters are specified.
+
+* If all parameters (module, provider, and version) are specified, the provided version for the given provider of the module is deleted.
```

Comment:
"provided version for the given provider" sounds a little wonky, since that base word kind of rotates its meaning 90° halfway through. How about "given version for the given provider?"

Reply:
yeah this was a mistake, good catch :)